src/app/app.module.ts

```diff
@@ -5,11 +5,13 @@ import { BrowserModule } from "@angular/platform-browser";
 import { AppRoutingModule } from "./app-routing.module";
 import { AppComponent } from "./app.component";
 import { ReactiveFormsModule } from "@angular/forms";
+import { SearchBarComponent } from "./search-bar/search-bar.component";
+import { SelectorComponent } from "./selector/selector.component";
 
 import { NgbDropdownModule } from "@ng-bootstrap/ng-bootstrap";
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent, SearchBarComponent, SelectorComponent],
```

Comment:
Why are you adding the components here if we don't have anything in the project to show?

I recommend you to have a module for each component so you can handle it's dependencies independently as you did with their stories and they can be then reused in the future in more than one place.

Reply:
To run the website it was required to add the dependencies in the module. I'm gonna search to check how to do a module for each component and push the new changes :)